lib/valueflow.h

```diff
@@ -229,7 +229,7 @@ namespace ValueFlow {
     std::string eitherTheConditionIsRedundant(const Token *condition);
 }
 
-const Variable *getLifetimeVariable(const Token *tok, ValueFlow::Value::ErrorPath &errorPath);
+const Variable *getLifetimeVariable(const Token *tok, ValueFlow::Value::ErrorPath &errorPath, int depth=20);
```

Comment:
A bit off-topic: is there a specific reason why those functions are declared within this header but outside the namespace?